js/hoot/control/utilities/ImportDirectory.js

```diff
@@ -168,10 +179,52 @@ Hoot.control.utilities.importdirectory = function(context) {
     * @desc gets file list from options in import directory file list
     **/
     var _getFilesList = function(){
-        var filesList = _.map(d3.select('#importDirectoryFilesList').selectAll('option')[0],function(opt){return opt.value;});
+        var filesList = _.map(d3.select('#importDirectoryFilesList').selectAll('option')[0],function(opt){return {value:opt.value,text:opt.text};});
         return filesList;
     };
 
+    var _checkForDescription = function(){
+        var mgcpCheck = d3.select('#importDirectorySchema').value().indexOf('MGCP');
+        var tdsCheck = d3.select('#importDirectorySchema').value().indexOf('TDS');
+        var cboxBool = mgcpCheck > -1 || tdsCheck > -1;
+        d3.select('.cboxAppendFCode').classed('hidden',!cboxBool).select('input').property('checked',false);
+        _getDescriptionList('MGCP');
```

Comment:
This arg sometimes needs to be TDS.

I'd almost rather this method set an internal variable that can be used instead of the code in _getTranslation()